tests/app/views/test_search.py

```diff
@@ -98,23 +98,32 @@ def test_should_get_services_next_page_of_services(self):
         with self.app.app_context():
             self.app.config['DM_SEARCH_PAGE_SIZE'] = '5'
             response = self.client.get(
-                '/index-to-create/services/search?q=serviceName&from=5')
+                '/index-to-create/services/search?q=serviceName')
```

Comment:
I don't understand how this tests anything interesting, if it doesn't start from a particular result. (i.e. it sounds like I'm as confused as you were, but don't we need to rename this test to make it more clear?)

Reply:
Yeah I'm frankly totally confused as to the point of it.

Reply:
Assuming test coverage doesn't decrease, let's remove it.

Reply:
Sure